sources/meta-srobo/recipes-robot/srobo-kit/srobo-kit.bb

```diff
@@ -1,5 +1,9 @@
 SUMMARY = "Package set to define a SR Kit"
 
+inherit useradd
+USERADD_PACKAGES = "srobo-kit"
+# password: tobor
+USERADD_PARAM:${PN} = "--create-home -r -s /bin/bash -G video,dialout -p '\$6\$m.5998lEWY2B.IxH\$nglPyX3BhXOPbBkeNNdJ4sLxFey5wJ543h1tNUpftZq8taJnYX19ewhTKz.HQpi.A33pjSzpPy/QxIOZ6PMeg/' robot"
```

Comment:
You're missing the wheel group here

Reply:
88f01ac

For some reason the package install of `srobo-kit` would fail if I added `wheel` to it.
The sudoers file shipped by the `sudo` recipe has a line for it but it's commented out unless the image has `pam` enabled as a feature.